test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -163,7 +163,6 @@ create_configs()
     echo "debug.cursor_reposition=0" >> $file_name  # WT-10594 - Not supported by older releases
     echo "debug.log_retention=0" >> $file_name      # WT-10434 - Not supported by older releases
     echo "debug.realloc_malloc=0" >> $file_name     # WT-10111 - Not supported by older releases
-    echo "huffman_key=0" >> $file_name              # WT-6893 - Not supported by newer releases
```

Comment:
The above WT-6893 goes to WT-6678 which says "As part of this ticket, we remove the support for only keys and retain the support for Huffman Encoding for values and documentation."

Question: Why does it mention documentation? Is it possible to store documentation in WiredTiger? If so, documentation of what?

Reply:
I think that's a phrasing thing. Rather than huffman encoding for (keys and documentation), it's (huffman encoding for keys) and documentation, which would refer to the stuff in the `docs/` directory.